apps/docs/pages/index.tsx

```diff
@@ -1,7 +1,165 @@
+import { Box, SimpleGrid, useColorModeValue } from "@chakra-ui/react";
+
+import { SectionCard } from "../features/section-card/SectionCard";
+import { SiteBody } from "../features/site-body/SiteBody";
+
 export default function Docs() {
   return (
-    <div>
-      <h1>Docs</h1>
-    </div>
+    <Box>
```

Comment:
Bare nyskjerrig! Er det meninga man skal kunne se denne siden et sted? Hvor må jeg gå for å se den? 🙂

Reply:
Den er ikke deployet noe sted enda. Ligger i backloggen :)

Reply:
Jeg mente lokalt, når jeg kjører opp prosjektet. Finner liksom ikke siden hehe

Reply:
Ah sånn ja. Om du drar ned denne branchen, så burde den finnes på `localhost:3000`. Meeen da bruker du plutselig Next istedenfor Remix for å rendre siden. Jeg jobber med å porte dette PRet over til den nye teknologistacken vår as we speak :)